multimedia/mjpg-streamer/files/mjpg-streamer.config

```diff
@@ -11,5 +11,6 @@ config mjpg-streamer 'core'
 	option led 'auto'
 	option www '/www/webcam'
 	option port '8080'
+	option listen_ip '192.168.1.1'
```

Comment:
Default should be **none** - not all routers use this subnet.

Reply:
Fixed, the default is now commented out.